File: 2466.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split("\n");

let b = parseInt(lines.shift());
let lista = lines.shift().split(' ');
let s = [];
while(true){
    for(var i = 0; i < b-1; i++){
        if(lista[i] == lista[i+1]){
            s.push(1);
        }
        else{
            s.push(-1);
        }
    }
    if(b == 2){
        break;
    }
    b-=1;
    lista = s;
    s = [];
}
s[0] == -1 ? console.log('branca') : console.log('preta');

File: 2466.py
b = int(input())
lista = input().split(' ')
s = []
while True:
    for i in range(0, b-1):
        if lista[i] == lista[i+1]:
            s+=[1]
        else:
            s+=[-1]
    if b == 2:
        break
    b-=1
    lista = s[:]
    s.clear()
print('branca') if s[0] == -1 else print('preta')
